store/file/ods_test.go

```diff
@@ -0,0 +1,133 @@
+//nolint:goconst
```

Comment:
This will be ineffective after upgrade to 1.59 golangci-lint and will be removed